### PR TITLE
Feature/iiwa 44

### PR DESCRIPTION
**Pull request by kirbur**
Created bar charts to be used in license card on dashboard

### CODE REVIEW

File: src/frontend-admin/src/components/Group/Group.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import styles from './Group.module.css'
+import cx from 'classnames' //allows use of more than 1 classname
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/Group/Group.tsx:3]: Nice! I've actually not used this library before - instead I've opted for util functions like this one: https://github.com/mtleonard120/react-ui-components/blob/master/src/utils/concatStyles/concatStyles.ts 

I would say go with it though :) Package looks good.

File: src/frontend-admin/src/components/Group/Group.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+//Types
+type JustifyType = 'start' | 'end' | 'between' | 'around' | 'even'
+type DirectionType = 'row' | 'column'
+
+interface IProps {
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/Group/Group.tsx]: I recommend making this name more descriptive to the component - like `IGroupProps`. The reason is that you may at some point want to export this interface for use elsewhere in the app and then `IProps` will not be descriptive enough.

File: src/frontend-admin/src/components/Group/Group.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    children: JSX.Element[],
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/Group/Group.tsx]: I somewhat regret using this type in my old repo (I'm assuming that's what I did). This is a rare case where `any` I think is the appropriate typing. The children can be lots of different things: elements, strings, conditional elements (like `{isModalVisible && <MyModal />}` and this particular typing is too restrictive to accommodate those

File: src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.module.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.barTitle {
+    text-transform: capitalize;
+    color: #fff;
+    font-size: 16px;
+    line-height: 25px;
+    text-align: left;
+    /*TODO: remove comment when we have font*/
+    /*font-family: CQL;*/
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.module.css]: Just as a heads up this font-family will eventually just be assigned to `body` in index.css (which is sort of the global css file) so you won't need to declare it in every component you write.

File: src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.test.tsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  const div = document.createElement('div');
+  ReactDOM.render(<HorizontalBarChart title={"Test"} amount={8} outOf={10} onClick={() => {}} />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+it('renders without crashing when amount > outOf', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<HorizontalBarChart title={"Test"} amount={8} outOf={4} onClick={() => {}}/>, div);
+  ReactDOM.unmountComponentAtNode(div);
+})
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.test.tsx:22]: Awesome! Love to see tests! Note that you can easily reuse props by doing something like 
```
const props = {
amount: 8,
onClick: () => {},
}
```
and then using them in your tests like `<HorizontalBarChart {...props} />` can save a lot of duplicated code when you have a lot of tests.

File: src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.tsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+import styles from './HorizontalBarChart.module.css'
+
+//Prop Type
+interface IProps {
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/reusables/HorizontalBarChart/HorizontalBarChart.tsx]: Same comment here on the interface name. If you export this interface you can actually use it in your test file when you create your `props` constant.

**Review by mtleonard120 — CHANGES_REQUESTED** [review]
Awesome - looking really good, just a few comments to address.

**Review by mcdowellalex — APPROVED** [review]

**Pull request merged**